src/core/include/openvino/core/preprocess/padding_mode.hpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+namespace ov {
+namespace preprocess {
+
+/// \brief An enum containing all supported padding modes in preprocessing
+enum class PaddingMode {
+    PAD_CONSTANT,           //!< padded values are taken from the value/values input
+    PAD_EDGE,               //!< Padded values are copied from the respective edge of the input data tensor
+    PAD_REFLECT,            //!< Padded values are computed by reflecting the input data tensor at the edge. Values on the edges are not duplicated.
+    PAD_SYMMETRIC           //!< Padded values are computed by reflecting the input data tensor at the edge, but values on the edges are duplicated.
+};
```

Comment:
maybe we can reuse https://github.com/openvinotoolkit/openvino/blob/ddf7c4491a82efa7c1df0d656fa2bd77b2ac3866/src/core/include/openvino/op/util/attr_types.hpp#L18 ?

Reply:
Hello. I noticed in resizing operation, enumeration class was redefined so I followed the practice(I think it's for namespace consideration but idk what it is really for).

Reply:
If really required make alias:
`using PaddingMode = ov::op::PadMode;`